float64[] gravity      # the estimated gravity acceleration
float64[] bias_gyr     # gyroscope bias
float64[] bias_acc     # accelerator bias
float64[] pos_end      # the estimated position at the end lidar point (world frame)
float64[] vel_end      # the estimated velocity at the end lidar point (world frame)
float64[] rot_end      # the estimated attitude at the end lidar point